10-JavaScript/exercises/extend-clock.md

```diff
@@ -0,0 +1,11 @@
+# Extend Digital Clock Project
+
+Earlier you used events & jQuery selectors to build a JavaScript powered digital clock.  Now we will extend it with jQuery click events and add some buttons to change how the clock displays.
+
+## Changing Color
+
+Add 3 buttons with text showing the names of colors.  Add an `onClick` event to the buttons which changes the text color of the clock to match the button.  An effective strategy would be to create css classes and use jQuery to switch the class the clock is styled with.
+
+## 24-hour vs 12-hour clocks
+
+Next add a 12-hour button which changes the clock to display in 12-hour time like 5:05:45pm, and a 24-hour button which changes the clock to display in 24-hour mode like 17:05:45.
```

Comment:
I think this is good for the basic requirement. I'd love to have an expanded optional/challenge for the button to toggle between the time clocks currently in use so it is only one single button.